src/main/java/com/gruzini/tennistico/repositories/ScoreRepository.java

```diff
@@ -2,6 +2,10 @@
 
 import com.gruzini.tennistico.domain.Score;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 public interface ScoreRepository extends JpaRepository<Score, Long> {
+    @Query("SELECT sc FROM scores sc JOIN FETCH sc.sets WHERE sc.id =:id")
```

Comment:
To trzeba było dodać, bo kaskadę mamy tylko na PERSIST, jo?

Reply:
nie do końca. Kaskady do wyciągania nie zrobisz przecież. To jest raczej alternatywa do dania fetch EAGER.

Reply:
Nie, że optuję za używaniem kaskad, chodziło mi bardziej o to, że jakbyśmy mieli ustawione np. CascadeType.ALL więc byłby też MERGE, to czy on by nie załatwił tego tematu. Twoje rozwiązanie jest spoko, pytałem raczej teoretycznie.

Reply:
ale od kiedy kaskada załatwia eager loading? Załatwia?

Reply:
Ok, nvm, faktycznie OneToMany jest defaultowo fetchowane LAZY.